.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,50 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
+  checkIfUiHasChanged:
```

Comment:
Let's use kebab-case for these job names. Also we should match the filename. We can rename this file and job to `check-if-ui-has-changed`,